packages/gatsby-cli/src/create-cli.js

```diff
@@ -209,8 +211,10 @@ function buildLocalCommands(cli, isLocalSite) {
         })
         .option(`prefix-paths`, {
           type: `boolean`,
-          default: false,
-          describe: `Serve site with link paths prefixed (if built with pathPrefix in your gatsby-config.js).`,
+          default: process.env.PREFIX_PATHS || false,
```

Comment:
Should we do the env dance?
```suggestion
        default: process.env.PREFIX_PATHS === 'true' || process.env.PREFIX_PATHS === '1',
```